BookingsApi/BookingsApi.UnitTests/Controllers/HearingsController/GetHearingsByGroupIdTests.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using BookingsApi.Contract.Responses;
+using BookingsApi.DAL.Queries;
+using BookingsApi.Domain;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+
+namespace BookingsApi.UnitTests.Controllers.HearingsController
+{
+    public class GetHearingsByGroupIdTests : HearingsControllerTests
+    {
+        [Test]
+        public async Task Should_Return_List_Of_Hearings_For_GroupId()
+        {
+            var groupId = Guid.NewGuid();
+            var hearing1 = GetHearing("123");
```

Comment:
To set this test up accurately, the case name should be 
'Case name Day 1 of 2' and 'Case name Day 2 of 2'